Helpjuice/ckeditor5
# [Editor] [sc-1901] Remove GHS feature for lists

## Describe the changes in this pull request

This PR fixes the issue where lists are suddenly split into several. That issue happens because we have enabled the GHS (General HTML Support) feature for lists. This isn't supported by CKEditor 5 as explained in their [docs](https://ckeditor.com/docs/ckeditor5/latest/features/html/general-html-support.html#known-issues):

![image](https://github.com/Helpjuice/ckeditor5/assets/28270050/5f0cf11b-9c31-42ab-94be-12fc0d81676d)

I also added a `.nvmrc` with the correct Node.JS version that should be used to build the CKEditor package. If you're experiencing issues on build, make sure that you're using the version that's on this file.

### Does this resolve an existing issue?

[[sc-1901](https://app.shortcut.com/helpjuice/story/1901/lists-are-not-displayed-correctly)]

### How should this be tested during the review?

1. Start a webserver on the ckeditor5 repo. You can do this by running the command below in the repo root:

```
ruby -run -e httpd . -p 8080
```

2. Navigate to `http://localtest.me:8080/packages/ckeditor5-build-helpjuice-v2/sample/index.html`.
3. Using the content from one of the articles in the card, join the `<ol>` elements that should have been one single element. You can also use this ~[HTML](https://drive.google.com/file/d/1H5iHSbgUi5z7H4iA9FnzpHSy89G-Vot1/view?usp=sharing) to test (it's from one of the articles with issues from the card)~ **NOTE**: The file doesn't exist anymore as of `2023-05-22`.
4. Open the source editor and paste the HTML there.
5. Close the source editor.

The content should be rendered without issues and the lists kept together. You can check this on the preview and by opening the source editor again.

Also, the list should behave as expected (including changing the list style through the editor button). Adding custom attributes to the `<ul>` or `<ol>` tag (like `style` or `class`) doesn't work and the attributes are ignored. On **master**, the attributes weren't added to the element, but to each of their `<li>` children (possibly causing issues).
